HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -7901,7 +7901,7 @@ Patch "Disable Umbrella Camera Lock-On" in "Camera" by "Hyper" does "Disables th
     );
 }
 
-Patch "Disable Vignette" in "Graphics" by "Hyper"
+Patch "Disable Vignette" in "Graphics" by "Hyper" does "Disables the vignette around the edges of the camera."
```

Comment:
Change to "Disables the edge darkening effect around the screen." Using the term "vignette" again is redundant and doesn't really explain what it is.